src/lib/protocols/credentials/CredentialService.ts

```diff
@@ -167,24 +169,25 @@ export class CredentialService extends EventEmitter {
    * @param credentialId Credential record ID
    * @param credentialResponseOptions
    */
-  public async createCredentialResponse(
+  public async createResponse(
```

Comment:
I'm not sure `createResponse` is the best name? I'm thinking of `createCredential`. Then we could also have `processCredential`

Reply:
Hmm, I understand. I also had a problem with this naming. It could be confused with an HTTP request/response. However, now we have consistency at least. Otherwise, we would need to change also other methods, I think. If you look at it from the protocol perspective, this naming is not actually that bad (`createRequest`, `createResponse`). Maybe, this message creation and processing stuff could evolve into something called `CredentialProtocol` calling service to do other tasks (saving credential record, creating protocol attachments, ...)